02 - Community Expansions/02.05 All Community Expansions/Themes/Micro Mike.md

```diff
@@ -0,0 +1,44 @@
+---
+aliases:
+- 
+tags: 
+- 
+publish: true
+---
+
+%% ----- Badges ----- %%
+
+![Downloads](https://img.shields.io/badge/downloads-98-573E7A?style=for-the-badge&logo=)
+![GitHub last commit](https://img.shields.io/github/last-commit/ThisTheThe/MicroMike?color=573E7A&label=last%20update&logo=github&style=for-the-badge)
+![GitHub issues by-label](https://img.shields.io/github/issues/ThisTheThe/MicroMike/help%20wanted?color=573E7A&logo=github&style=for-the-badge) 
+![GitHub Repo stars](https://img.shields.io/github/stars/ThisTheThe/MicroMike?color=573E7A&logo=github&style=for-the-badge)
+
+%% ----- Badges ----- %%
+
+%% ----- Do not edit this section ----- %%
+
+# Micro Mike
+
+Repository: [GitHub](https://github.com/ThisTheThe/MicroMike)
+Designed by: [[ThisTheThe]]
+Modes: [[Dark-mode themes|dark]]
+
+
+
+![screenshot](https://github.com/ThisTheThe/MicroMike/raw/master/screenshotThumb.png)
```

Comment:
This image does not exist:
https://github.com/ThisTheThe/MicroMike/raw/master/screenshotThumb.png

This one does:
https://github.com/ThisTheThe/MicroMike/raw/main/screenshotThumb.png

The same applies to several other pages here.

Reply:
Logged in https://github.com/obsidian-community/obsidian-hub/issues/536